code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -875,7 +875,8 @@ func (fsh *StorageHandler) CreateDir(ctx context.Context, r *http.Request) (*blo
 		return nil, common.NewError("invalid_parameters", "Invalid dir path passed")
 	}
 
-	exisitingRef := fsh.checkIfFileAlreadyExists(ctx, allocationID, dirPath)
+	// Update checkIfFileAlreadyExists to checkIfFileRefIDAlreadyExists
+	exisitingRef := fsh.checkIfFileRefIDAlreadyExists(ctx, allocationID, dirPath)
```

Comment:
We can directly call `reference.GetReferenceID(ctx, allocationID, path)`. No need to call `checkIfFileRefIDAlreadyExists`.